README.md

```diff
@@ -89,6 +97,9 @@ LongCounter counter = meter
     .build();
 
 counter.add(123, Attributes.of(stringKey("job-type"), "print-receipt"));
+
+// Sleep for some seconds so the PeriodicMetricReader has time to finish exporting
+Thread.sleep(5000);
```

Comment:
Is this needed? :thinking: 
I thought that there may be some default shutdown hook that makes sure metrics are flushed before exiting :thinking:

Reply:
Without the sleep I didn't manage to see any metrics. I also tried `meterProvider.shutdown()` with no avail.